src/commands/Testing/hax.ts

```diff
@@ -23,19 +26,192 @@ export default class extends BotCommand {
 		) {
 			return;
 		}
-
 		const paths = Object.values(Skills).map(sk => `skills.${sk.id}`);
-
-		msg.author.settings.update(paths.map(path => [path, 14_000_000]));
-		msg.author.settings.update(UserSettings.GP, 1_000_000_000);
-		msg.author.settings.update(UserSettings.QP, 250);
-		const loot = {};
+		await msg.author.settings.update(paths.map(path => [path, 13_034_431]));
+		await msg.author.settings.update(UserSettings.GP, 2_147_483_647);
+		await msg.author.settings.update(UserSettings.QP, MAX_QP);
+		await msg.author.settings.update(UserSettings.SacrificedValue, 10_000_000_000);
+		if (!msg.author.minionIsBusy) {
+			const rangeGear = {
+				'2h': { item: itemID('Heavy ballista'), quantity: 1 },
+				head: { item: itemID('Armadyl helmet'), quantity: 1 },
+				neck: { item: itemID('Necklace of anguish'), quantity: 1 },
+				body: { item: itemID('Armadyl chestplate'), quantity: 1 },
+				legs: { item: itemID('Armadyl chainskirt'), quantity: 1 },
+				feet: { item: itemID('Pegasian boots'), quantity: 1 },
+				cape: { item: itemID("Ava's assembler"), quantity: 1 },
+				shield: null,
+				hands: { item: itemID('Barrows gloves'), quantity: 1 },
+				ring: { item: itemID('Archers ring (i)'), quantity: 1 },
+				weapon: null,
+				ammo: { item: itemID('Onyx bolts (e)'), quantity: 99 }
```

Comment:
These could be made much cleaner with the new `constructGearSetup` thing I added. Feel free to move this function into `lib/minions/gear` or somewhere if we're using it outside tests now.